README.md

```diff
@@ -1,3 +1,33 @@
 # Go client and cli for [outline][def]
 
-[def]: https://www.getoutline.com/
\ No newline at end of file
+[def]: https://www.getoutline.com/
+
+
+
+# Usage
+
+
+## Client
+Getting a single document by its id:
+```golang
+cl := outline.New()
```

Comment:
`cl`? What about `ol` (for `outline`) or `client`... I don't like those abbreviations 😅

Reply:
Identifier names should be proportional to their scope. The farther they are used from declaration the more pronounced their names should be. But here IMO the name is apt as per usage scenario. This how you will find var names in most go codebase :laughing: I suggest: https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_choose_identifiers_for_clarity_not_brevity